examples/inverse-kinematics.py

```diff
@@ -20,15 +20,16 @@
 i=0
 while True:
     pinocchio.forwardKinematics(model,data,q)
-    dMi = oMdes.actInv(data.oMi[JOINT_ID])
-    err = pinocchio.log(dMi).vector
+    iMd = data.oMi[JOINT_ID].actInv(oMdes)
+    err = pinocchio.log(iMd).vector  # in joint frame
     if norm(err) < eps:
         success = True
         break
     if i >= IT_MAX:
         success = False
         break
-    J = pinocchio.computeJointJacobian(model,data,q,JOINT_ID)
+    J = pinocchio.computeJointJacobian(model,data,q,JOINT_ID)  # in joint frame
+    J = -pinocchio.Jlog6(iMd.inverse()) @ J
```

Comment:
@stephane-caron This is why ROS Melodic CI is failing - the `@` operator is Python3, Melodic is Python2.7. `np.dot` should fix it
```suggestion
    J = -np.dot(pinocchio.Jlog6(iMd.inverse()), J)
```

Reply:
workarounded in #1966